back/boxtribute_server/business_logic/box_transfer/shipment/mutations.py

```diff
@@ -17,23 +15,13 @@ def resolve_create_shipment(*_, creation_input):
         permission="shipment:create",
         base_ids=[creation_input["source_base_id"], creation_input["target_base_id"]],
```

Comment:
should this not be `base_id=creation_input["source_base_id"]` to ensure that the use has the permission at the source base id? Only users at the source organisation should be able to create a shipment, right?

Reply:
thanks, fixed it